### DISCUSSION TITLE
Help to fill form

### DISCUSSION THREAD

**Discussion by Shokoaviv** [Budibase/budibase#3265] [Help] [answered]
![Screen Shot 2021-11-06 at 5 16 19 PM](https://user-images.githubusercontent.com/3799973/140614680-b28f643a-fdf4-46b9-be21-a83421e7ddcf.png)
 
Hi 

Need Help,p to fill up this form/ Anybody?

**Comment by mjashanks** [comment, marked as answer]
These are some standard SMTP setup details, that you should get from your email provider. For example, I use [Postmark](https://postmarkapp.com/). I can get all of theses details inside the postmark platform: https://postmarkapp.com/developer/user-guide/send-email-with-smtp
